centreon/www/class/centreonInstance.class.php

```diff
@@ -59,6 +63,20 @@ public function __construct($db, $dbo = null)
         $this->initParams();
     }
 
+    /**
+     * @param CentreonDB $db
+     * @param CentreonDB|null $dbo
+     * @return CentreonInstance
+     * @throws PDOException
+     */
+    public static function getInstance(CentreonDB $db, ?CentreonDB $dbo = null): CentreonInstance
+    {
+        if (self::$staticInstance === null) {
+            self::$staticInstance = new CentreonInstance($db, $dbo);
+        }
+        return self::$staticInstance;
```

Comment:
`new self` + coallesce 
```suggestion
        return self::$staticInstance ??= new self($db, $dbo);
```